include/kodlab_mjbots_sdk/low_pass_filter.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file low_pass_filter.h
+ * @author Lucien C. Peach (peach@seas.upenn.edu)
+ * @brief A discrete-time low-pass filter class.
+ * @date 9/10/2022
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include "kodlab_mjbots_sdk/filter.h"
+#include "Eigen/Geometry"
+
+namespace kodlab
+{
+
+class LowPassFilter : public Filter {
```

Comment:
**[General]** Same comment as in `Filter`.  This should probably be a [class template](https://en.cppreference.com/w/cpp/language/class_template).  
Note that, for this class, both the definition and implementation would have to live in the header file if you make this a class template.  See [here](https://isocpp.org/wiki/faq/templates#templates-defn-vs-decl) for more information.